test/integration/elsa/consumer_test.exs

```diff
@@ -58,6 +58,68 @@ defmodule Elsa.ConsumerTest do
     Supervisor.stop(pid)
   end
 
+  test "Elsa.Consumer will still send messages after reassignment" do
```

Comment:
I added this test because I'm about to mess with the code that handles rejoining consumer groups.  Wanted to have a baseline on the existing codebase so I can tell if/where I break things.